### DISCUSSION TITLE
How To Get All Edges Connected To A Node?

### DISCUSSION THREAD

**Discussion by karahanoguzhan** [bcakmakoglu/vue-flow#446] [Q&A] [answered]
How can I list all edges connected to a specific node?

**Comment by bcakmakoglu** [comment, marked as answer]
[Use the `getConnectedEdges` utility.](https://vueflow.dev/typedocs/functions/getConnectedEdges.html)

```ts
const connectedEdges = getConnectedEdges([node], edges)
```
